lib/generators/active_record/devise_two_factor_generator.rb

```diff
@@ -6,7 +8,10 @@ class DeviseTwoFactorGenerator < ActiveRecord::Generators::Base
       source_root File.expand_path('../templates', __FILE__)
 
       def copy_devise_migration
-        migration_template 'migration.rb', "db/migrate/devise_two_factor_add_to_#{table_name}.rb"
+        migration_template 'migration.rb',
+                           "db/migrate/devise_two_factor_add_to_#{table_name}.rb"
+        migration_template 'add_last_successful_otp_at.rb',
```

Comment:
What happens to somebody that already has the first migration complete? Do they just re-run and migration_template does some magic?

Reply:
Yes